site/_docs/reference.md

```diff
@@ -2861,7 +2861,7 @@ In the following:
 | b | TIME_DIFF(time, time2, timeUnit)               | Returns the whole number of *timeUnit* between *time* and *time2*
 | b | TIME_SUB(time, interval)                       | Returns the TIME value that is *interval* before *time*
 | b | TIME_TRUNC(time, timeUnit)                     | Truncates *time* to the granularity of *timeUnit*, rounding to the beginning of the unit
-| o p | TO_CHAR(timestamp, format)                 | Converts *timestamp* to a string using the format *format*
+| o p | TO_CHAR(timestamp, format)                   | Converts *timestamp* to a string using the format *format*
```

Comment:
Sorry, some formatting issues in the previous PR have been fixed.
https://github.com/apache/calcite/pull/3645